subsys/bluetooth/controller/hci_driver.c

```diff
@@ -684,6 +683,9 @@ static int configure_memory_usage(void)
 	}
 #endif
 
+#if defined(CONFIG_BT_PERIPHERAL) || \
+	defined(CONFIG_BT_CENTRAL) || \
```

Comment:
Why do we have tests that set both CONFIG_BT_CENTRAL=n and CONFIG_BT_PERIPHERAL=n ? Or, what is the use-case? I guess there's some reason, but I also see that we don't have an SDC variant which supports it

Reply:
a simple broadcaster is a usecase for this.
Also quite some of our tests enable BT, but doesnt enable any roles at all.. for example simple flash access tests.